tests/arp/test_neighbor_mac_noptf.py

```diff
@@ -37,6 +38,8 @@ def restoreDutConfig(self, duthosts, enum_rand_one_per_hwsku_frontend_hostname):
                 None
         """
         duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+	duthost.command("sudo config bgp shutdown all")
+	time.sleep(120)
```

Comment:
Please use white space instead of tabs.

Also, please change the time.sleep to wait_until bgp sessions are all down. You can still wait up to 2 minutes. but the wait will end as soon as conditions are met.

Reply:
Please check this example: https://github.com/Azure/sonic-mgmt/blob/master/tests/bgp/conftest.py#L106 but use pytest_assert instead.

Reply:
@yxieca What Alex meant is even if the BGP session tears down, the entire changes would not have been propagated to SWSS. SWSS might be in the process of removing the routes due to tear down and thus wouldn't be able to handle neighbor mac notifications. This is the reason we require 120 sec delay to accommodate SWSS processing and not just BGP.

Reply:
@dgsudharsan that is a good point. But the question is that how can we know a definitive time when the routes are all removed? If that is the condition for test to move forward.
My concern is that (1). Test uniformly wait for a long time. (2). The long time might not be enough in some cases.

Reply:
@yxieca  We are not aware of a definitive wait time or indicator from SWSS currently. Currently most of the the wait (like config reload) waits for 120 seconds and that's the reason the number is chosen. Are there any other suggestions from your end to have a definitive wait based on some indicators?

Reply:
@dgsudharsan  how about this approach here: https://github.com/Azure/sonic-mgmt/blob/master/tests/route/test_route_perf.py#L174

What you want is wait until there is no route, right?